app/templates/courses/teacher-courses-view.jade

```diff
@@ -61,11 +61,14 @@ block content
                         each level, levelIndex in campaign.getLevels().models
                           - var levelNumber = campaign.getLevelNumber(level.get('original'), levelIndex + 1)
                           option(value=level.get('slug'))
-                            span
-                              = levelNumber
-                            span.spr
-                              | .
-                            = " "
+                            if (/challenge/i).test(levelNumber)
+                              =$.t('courses.challenge') + ' '
```

Comment:
It would be better to have this based on if `level.get('assessment')`. Then `levelNumber` need never be `"Challenge"`, which is a brittle way to pass on that information anyway.

Reply:
Updated to not use the regex.

I think the proper solution long term is to refactor all the places that do basically this logic (levelNumber + '. ' + levelName) and use a single helper method to generate that whole string, so we can get 'Challenge: LevelName' out of it.

For now, I'm going to still have levelNumber be "Challenge" since it basically fits with level numbers being allowed to be eg "7a", and some views want that string.